### ISSUE TITLE
Why does OSM connect to 104.131.68.237 ?

### ISSUE THREAD

**Issue by gerroon** [eolwral/OSMonitor#75]
HI

I see that OSM always connecting to  ip above. It is 443 so encrypted, I can decrypt that ssl. However I wanted to ask the developer first about this particular connection before going ahead. If this is a map functionality why would it need to connect to  Digital Ocean in the first place? 

Is this data leak?

**Comment by eolwral** [comment]
Hi gerroon,

It is an endpoint of WHOIS function, when you want to query, it will connection to the server and get result, I rented a VPS on DigitalOcean and run the service.

This is URL for the endpoint
https://github.com/eolwral/OSMonitor/blob/e23d06d9cec908bafc52b9a42f57c3d7ac59b58c/app/src/main/java/com/eolwral/osmonitor/ui/ConnectionFragment.java#L607

If you are interested, you can see the source code of the endpoint from another repo.
https://github.com/eolwral/proxy-whoisapi

**Comment by gerroon** [comment]
Hi

Thanks for the follow up. What kind of information is transferred to your VPS? Is it only the IP in question?

**Comment by eolwral** [comment]
Yes, it only sends the IP to query, I don't gather device specifics. However, HTTP's user-agent header may leak the your device name, but I can promise here, I won't collect the device name, I think it is a privacy issue. :)

The API is available via the browse,  you can access via simple URL and get JSON result. 
>  https://osmonitor.mobi/WhoisAPI?IP=111.111.111.111

Actually my API endpoint is a cache proxy, I feel I don't have right to share the information twice, if you want to use, I would suggest you to use the original service, it is a free service for non-commercial purpose, please check following link.
> http://ip-api.com/docs/#usage_limits

**Comment by gerroon** [comment]
@eolwral 

Thanks for the follow up.
